### ISSUE TITLE
Selecting "Hide new user announcements" disables the entire app

### ISSUE THREAD

**Issue by AbleLincoln** [hatnote/listen-to-wikipedia#27]

**Comment by finetype** [comment]
Yeah, huge pain, because those string swells ruin the experience.

**Comment by mahmoud** [comment]
Ah, yes, as long as we're referring to the web app, there's a small bug in the JavaScript. You just need to re-enable your selected language. (i.e., re-click English Wikipedia after clicking "disable new user announcements")
